pallets/staking/src/inflation.rs

```diff
@@ -76,9 +78,6 @@ mod test {
     #[test]
     fn npos_curve_is_sensible() {
         const YEAR: u64 = 365 * 24 * 60 * 60 * 1000;
```

Comment:
I think building the constants off each other improves readability.
```suggestion
        const YEAR: u64 = 365 * DAY;
```

Reply:
The goal here is to minimize the diff to substrate [3.0.0](https://github.com/paritytech/substrate/blob/v3.0.0/frame/staking/src/lib.rs) so keeping as-is. (Although I agree with you...)